nav2_simple_commander/nav2_simple_commander/costmap_2d.py

```diff
@@ -1,6 +1,7 @@
 #! /usr/bin/env python3
 # Copyright 2021 Samsung Research America
 # Copyright 2022 Stevedan Ogochukwu Omodolor
+# Copyright 2022 Jaehun Jackson Kim
```

Comment:
If you're doing this on behalf of a company, you should list your company instead, just FYI :smile:

Reply:
Okay, Thank you for your information, It is my first time, So I missed it. I will change 😊

Reply:
It is not for the company, So I think don't need to change